channels/chan_simpleusb.c

```diff
@@ -2260,6 +2260,13 @@ static struct ast_frame *simpleusb_read(struct ast_channel *c)
 		sd = 1;
 	}
 	o->rx_ctcss_active = sd;
```

Comment:
Shouldn't this assignment follow the new code?

Reply:
I left that so that the live display would show the actual signal being received.  We may have a cos activation and that would be displayed, however, if the options are set to 'no', the ptt would not be keyed.

Setting both options to 'no' is a real unique situation.  It would be a case where the device can only transmit.